core/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -108,6 +108,23 @@ class FormatOps(val tree: Tree, val initStyle: ScalafmtConfig) {
     (packages.result(), imports.result(), arguments.toMap, optional)
   }
 
+  object CtorModifier {
```

Comment:
Can we move this to https://github.com/scalameta/scalafmt/blob/master/core/src/main/scala/org/scalafmt/util/TreeClasses.scala ?
 
The `@classifier` macro annotation creates a `Classifiable` type-class instance, which makes it possible to use the `Token/Tree.is[T]` syntax. You can probably avoid the macro annotation by writing some boilerplate yourself, but I haven't looked at the expanded code. It's up to yourself.

The (undocumented) general rule where I keep utilities is

- FormatOps: anything that requires access to the fields in FormatOps, which is custom for each source file.
- Token/TreeOps: anything that is not require fields in FormatOps, agnostic to the source file being formatted.
- Token/TreeClasses: classifiers like this.

I should probably add this to the contributing docs.